containerd-shim-spin/src/engine.rs

```diff
@@ -224,8 +229,11 @@ impl SpinEngine {
                         .context("failed to build spin trigger")?;
 
                     info!(" >>> running spin http trigger");
+                    let address_str = env::var(SPIN_HTTP_LISTEN_ADDR_ENV)
+                        .unwrap_or_else(|_| SPIN_ADDR_DEFAULT.to_string());
```

Comment:
in some places I've seen erroring out if user provided an invalid input instead of ignoring it. but no strong opinions about it. what do you think? may be print a warning somewhere?

Reply:
Here we are just checking if the user set the env var at all. We will error out in the next line (`parse_addr`) if it is invalid.